ios/FluentUI/Badge Field/BadgeView.swift

```diff
@@ -11,11 +11,14 @@ open class BadgeViewDataSource: NSObject {
     @objc open var text: String
     @objc open var style: BadgeView.Style
     @objc open var size: BadgeView.Size
+    @objc open var avatar: MSFAvatar?
 
-    @objc public init(text: String, style: BadgeView.Style = .default, size: BadgeView.Size = .medium) {
+    @objc public init(text: String, style: BadgeView.Style = .default, size: BadgeView.Size = .medium, avatar: MSFAvatar? = nil) {
```

Comment:
As it was already pointed out, it seems a better approach to have the badge generic enough to accept a UIView instead of fixing it to the avatar.
The 2 main benefits of that is that you don't create a dependency from the BadgeView to the Avatar (the podspec file would need to be updated with this approach), and still get to add other types as this leading view (such as UIImageView, UIImage, etc...).

Reply:
Updated to `UIView?`